linera-views/src/reentrant_collection_view.rs

```diff
@@ -326,6 +326,168 @@ where
     }
 }
 
+impl<C, W> ReentrantByteCollectionView<C, W>
+where
+    C: Context + Send + Clone + 'static,
+    ViewError: From<C::Error>,
+    W: View<C> + Send + Sync + 'static,
+{
+    /// Load multiple entries for write at once.
+    /// The entries in short_keys have to be all distincts.
+    /// ```rust
+    /// # tokio_test::block_on(async {
+    /// # use linera_views::memory::{create_test_context, MemoryContext};
+    /// # use linera_views::reentrant_collection_view::ReentrantByteCollectionView;
+    /// # use linera_views::register_view::RegisterView;
+    /// # use crate::linera_views::views::View;
+    /// # let context = create_test_context();
+    ///   let mut view : ReentrantByteCollectionView<_, RegisterView<_,String>> = ReentrantByteCollectionView::load(context).await.unwrap();
+    ///   let short_keys = vec![vec![0,1],vec![2,3]];
```

Comment:
nit: missing spaces after `,`

Sadly, rustfmt doesn't work here.

Reply:
Ok, done (and also to collection_view).